### DISCUSSION TITLE
reset function in compliance script

### DISCUSSION THREAD

**Discussion by n-schoofs** [usnistgov/macos_security#286] [Q&A] [answered]
Hi there

Looking at the script logic, you'd be able to reset the .plist with the reset() function. However, I can't seem to pass on the -reset flag. I'm using Jamf Pro, running the script through a Policy and passing the -reset flag like I pass the -check or -cfc flags. Any ideas what I might be doing wrong? :-)

**Comment by jmahlman** [comment]
Are you using `--reset` ?

**Reply by n-schoofs** [reply]
Yup. Tried it as `-reset` and `--reset`. When I force run the policy with `sudo jamf policy -id XX` nothing happens. When I quit the `jamf policy` command it appears that the compliance script menu was shown and waiting for input. I add the flag in the first parameter ("Parameter 4") placeholder in the Script options of the Policy.

**Reply by tonyyo11** [reply]
Running into the exact same problem today after having the need to reset mSCP. I have 0% complete for my policy attempting to reset the project. I believe it’s as you say @n-schoofs - the script is just sitting there waiting for additional input. Will have to just manually reset it with a separate script to delete the PLIST file

**Reply by golbiga** [reply, marked as answer]
You need to pass `--check` in $4 and `--reset` in $5

**Reply by n-schoofs** [reply]
This works, thanks!
